packages/next/lib/check-lock-file.ts

```diff
@@ -0,0 +1,19 @@
+import lock from 'proper-lockfile'
+import { mkdirSync } from 'fs'
+
+export default function checkLockFile(distDir: string) {
+  try {
+    mkdirSync(distDir)
```

Comment:
Do we need to create the directory for them?

Reply:
Yeah, `proper-lockfile` requires the file/directory exists before creating the lock for it

Reply:
Is ignoring this failure the right thing to do here? I feel like we should skip the lock file if this fails.

Reply:
I updated it to only ignore an `EEXIST` error since this error is alright and other errors should be fatal